server/methods.js

```diff
@@ -364,23 +378,30 @@ Meteor.methods({
   soundcloudAudioSearchList: function(query, option, page) {
     var res;
     var items, nextPage;
+    var path = 'tracks';
     check(query, String);
+
+    if (query.indexOf('soundcloud.com') !==-1) {
+      path = 'resolve'
```

Comment:
if/else would be clearer here than setting `path` above and then sometimes updating it here